expat/doc/reference.html

```diff
@@ -1112,12 +1113,40 @@ <h4 id="XML_Parse">XML_Parse</h4>
 <code>isFinal</code> parameter informs the parser that this is the last
 piece of the document. Frequently, the last piece is empty (i.e.
 <code>len</code> is zero.)
+</p>
+
+<p>
 If a parse error occurred, it returns <code>XML_STATUS_ERROR</code>.
 Otherwise it returns <code>XML_STATUS_OK</code> value.
 Note that regardless of the return value, there is no guarantee that all
 provided input has been parsed; only after <a href="#isFinal">the
 concluding call</a> will all handler callbacks and parsing errors have
 happened.
+</p>
+
+<p>
+Function <code>XML_Parse</code> is mostly a convenience wrapper pairing
```

Comment:
nit: remove leading "Function"?

Reply:
It felt like a feature to say function here. Could you help me understand how dropping the word helps in your view?

Reply:
It just felt like weird grammar to me. "The XML_Parse function..." feels better, and "The function XML_Parse ..." somewhat less so. But I felt identifying it as a function was superfluous anyway, so the simplest suggestion was to remove it.

Reply:
@Snild-Sony alright, thanks for elaborating! :pray: